bridgestyle/arcgis/togeostyler.py

```diff
@@ -797,6 +798,10 @@ def _extractFillOpacity(symbolLayers):
     return 1.0
 
 
+def _extractFontWeight(textSymbol):
+    return "bold" if textSymbol.get("fontStyleName") == "Bold" else "regular"
```

Comment:
Before it was `Regular`, with an uppercase `R`.
Is it wanted to have it lowercase now ?

Reply:
Yes, because in the SLD spec it is lower case (https://docs.geoserver.org/maintain/en/user/styling/sld/reference/textsymbolizer.html#font). The value was parsed but never used. Actually, it seems to be case insensitive in GeoServer, but I find it better to follow the spec.
And your comment made me realized that it should be `normal` not `regular` (I got it right in the commit and PR description and it was working, but for the wrong reasons.. always good to have a second pair of eyes, especially without tests)